GymApp/GymAppApi/Models/CouchVisitor.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace GymAppApi.Models
+{
+    public class CouchVisitor
+    {
+        public int CouchId { get; set; }
+        public Couch  Couch { get; set; }
+        public int VisitorId { get; set; }
+        public Visitor visitor { get; set; }
```

Comment:
Separate attributes from links. Add "= null!" or "?" according to relationships in database